language/sample-apps/chat-flask-cloudrun/requirements.txt

```diff
@@ -0,0 +1,2 @@
+google-cloud-aiplatform==1.27.1
```

Comment:
Can you see if you can update with version >= 1.31.0?

Reply:
Done